eva-server/src/test/java/uk/ac/ebi/eva/server/repository/VariantEntityRepositoryTest.java

```diff
@@ -124,14 +129,72 @@ public void testVariantRegionIsFoundMultiple(){
 
     @Test
     public void testNonExistentVariantRegionIsNotFound(){
-        String chr = "20";
+        String chr = "11";
         int start = 61098;
         int end = 60916;
-        final List<VariantEntity> variantEntityList = variantEntityRepository.findByChrAndStartWithMarginAndEndWithMargin(chr, start, end);
+        List<VariantEntity> variantEntityList =
+                variantEntityRepository.findByRegionAndComplexFilters(chr, start, end, new ArrayList<>(), null, null,
+                                                                      null, null, null, null, new ArrayList<>());
         assertNotNull(variantEntityList);
         assertTrue(variantEntityList.size() == 0);
     }
 
+    private void testFiltersHelper(String chr, int start, int end, List<String> consequenceType,
```

Comment:
According to our code style guide, private methods must be placed after the methods that use them.